data.md

```diff
@@ -246,6 +256,9 @@ Operator `_++_` implements an append operator for sort `ValStack`.
 
     rule #drop(.ValTypes,   VALSTACK)                       => VALSTACK
     rule #drop(TYPE VTYPES, < TYPE > VAL:Number : VALSTACK) => #drop(VTYPES, VALSTACK)
+
+    rule #revs(.ValStack) => .ValStack
+    rule #revs(V : VS)    => #revs(VS) ++ V : .ValStack
```

Comment:
This is a very slow way to reverse the list, actually is quadratic. Can we make it tail-recursive instead?

```
syntax ValStack ::= #revs ( ValStack ) | #revs ( ValStack , ValStack ) [klabel(#revsAux)]
rule #revs(VS) => #revs(VS, .ValStack)
rule #revs(.ValStack, VS') => VS'
rule #revs(V : VS, VS') => #revs(VS, V : VS')
```

Then it's linear and won't blow up the memory usage of the OCaml and LLVM backend. Don't forget the extra klabel.